api/services/Issues/getAnnotation.js

```diff
@@ -6,10 +6,28 @@
  */
 
 module.exports = function getAnnotation(description) {
+    const annotationMatchers = [
+        {
+            rex: /@w-\d+@/gi,
+            replace: /@/g
+        },
+        {
+            rex: /GUS-W-\d+/g,
```

Comment:
How about `\b` at the start and end of this rex, so that only complete words are matched?

```
/\bGUS-W-\d+\b/g
```